internal/node/node.go

```diff
@@ -130,7 +130,7 @@ func (n *Node) GetObject(rootCtx context.Context, hash string) (data string, err
 // TODO: add concurrency
 func (n *Node) pingContacts(ctx context.Context, me *contact.Contact, targetIps []string) (*contact.Contact, error) {
 	for _, targetIp := range targetIps {
-		contact, err := n.Client.SendPing(ctx, me, targetIp+":50051") // TODO fix this port
```

Comment:
can we change targetIps in the func header? The function is called "pingContacts", so i expect to pass multiple contacts and not ip addresses. Also, we then dont need to fix the port, but can easily use the contact address, which includes ip+port.